config/locales/de.yml

```diff
@@ -0,0 +1,230 @@
+---
+de:
+  message:
+    message:
+      message_date: "%{time_ago} vorher"
```

Comment:
Das klingt irgendwie holprig. Warum nicht "vor 10 Stunden"?

Reply:
Ja, bitte

Reply:
Weil du dann die default Übersetzungen anpassen musst. Sonst steht da "Vor 10 Tage".

Es gibt einen [Thread auf Github](https://github.com/svenfuchs/rails-i18n/issues/217) wo das Problem beschrieben ist. Als Lösung kann man einen `scope` angeben. Das habe ich dann auch einen commit vorher [hier](https://github.com/tactilenews/100eyes/commit/4d40e71cdc9c9578996fc326ffc3216b30eb62bf#diff-ec3964c7889ef457fe4682bc9427032eL6) und [hier](https://github.com/tactilenews/100eyes/commit/4d40e71cdc9c9578996fc326ffc3216b30eb62bf#diff-2c99272ca3b6d3c55bd5c1c3f14f077eR75) gemacht. Dann habe ich mir es anders überlegt, weil ich es auch für alle anderen Zeiteinheiten (Minuten, Sekunden, Monate) anpassen müsste.

Reply:
Ah, ok – ergibt Sinn. Wenn’s nicht zu dramatisch ist, würde ich das dann auch erstmal so lassen, wie es jetzt ist, @drjakob?

@roschaefer: Brauchen wir dann überhaupt [diese ganzen Übersetzungen](https://github.com/tactilenews/100eyes/pull/61/files#diff-2c99272ca3b6d3c55bd5c1c3f14f077eR18) noch in unserer Locale-Datei? Oder sind die schon per Default in Rails vorhanden?

Reply:
@tillprochaska das ist copy+paste von [hier](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/de.yml) ich verstehe gar nicht, warum ich die überhaupt hinzufügen musste und warum die nicht per default vorhanden sind. Ich würde sie so wie sie sind, drin lassen.